File: morris/morris_game/Position.py
'''
Created on Oct 3, 2015

@author: speng
'''

class Position(object):
    '''
    classdocs
    '''
    

    def __init__(self, pos, player):
        '''
        Constructor
        '''
        self.coord = pos
        self.player = player
        self.up = None
        self.down = None
        self.left = None
        self.right = None

File: src/Position.java

class Position {
	private String coord;
	private Player player;
	private Position up;
	private Position down;
	private Position left;
	private Position right;
	
	Position(String pos, Player player) {
		this.coord = pos;
		this.player = player;
		this.up = null;
		this.down = null;
		this.left = null;
		this.right = null;
	}
	
	public void setUp(Position position)
	{
		this.up = position;
	}
	
	public void setDown(Position position)
	{
		this.down = position;
	}
	
	public void setRight(Position position)
	{
		this.right = position;
	}
	
	public void setLeft(Position position)
	{
		this.left = position;
	}
}
